### PR TITLE
chore: Update readme

### PR DESCRIPTION
**Pull request by oed**
Features the following changes:
* Clean up the readme (tnx @michaelsena)
* Move Development section to separate file
* Rename `alpha` releases to `rc` (release clients)
* Remove docs for http-api (moved to general doc site: https://github.com/ceramicnetwork/docs/pull/8)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "test": "lerna exec npm t",
     "build": "lerna run build",
     "docs": "./node_modules/.bin/typedoc",
-    "publish:next": "lerna publish --no-push --conventional-prerelease --dist-tag next",
+    "publish:release-client": "lerna publish --no-push --conventional-prerelease --dist-tag next --preid rc",
```
Comment (stbrody) [inline package.json:9]: RC usually refers to "release candidate", I've never actually heard the term "release client" before...

**Review by stbrody — COMMENTED** [review]
looks good, main comment is renaming "release client" to "release candidate" throughout

File: package.json (hunk 1 above)
Comment (oed) [reply package.json:9]: my bad!

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+| [@ceramicnetwork/doctype-tile](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/doctype-tile) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/doctype-tile)](https://www.npmjs.com/package/@ceramicnetwork/doctype-tile) | Interface for tile doctype |
+| [@ceramicnetwork/doctype-tile-handler](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/doctype-tile-handler) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/doctype-tile-handler)](https://www.npmjs.com/package/@ceramicnetwork/doctype-tile-handler) | Handler for tile doctype |
+| [@ceramicnetwork/http-client](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/http-client) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/http-client)](https://www.npmjs.com/package/@ceramicnetwork/http-client) | HTTP client that can interact with a remote Ceramic daemon |
+| [@ceramicnetwork/pinning-aggregation](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/pinning-aggregation) | [![npm](https://img.shields.io/npm/v/@ceramicnetwork/pinning-aggregation)](https://www.npmjs.com/package/@ceramicnetwork/pinning-aggregation) |  |
```
Comment (stbrody) [inline DEVELOPMENT.md:45]: Can we also get descriptions for the pinning packages?  Sorry forgot to say this in last review round

**Review by stbrody — APPROVED** [review]

**Pull request merged**